fabric-data-attachment-api-v1/src/main/java/net/fabricmc/fabric/api/attachment/v1/AttachmentTarget.java

```diff
@@ -219,6 +221,20 @@ default <A> A removeAttached(AttachmentType<A> type) {
 		return setAttached(type, null);
 	}
 
+	/**
+	 * Provides an {@link Event} that lets the listener know when the target has its {@link AttachmentType} attachment changed.
+	 *
+	 * <p>
+	 * The event is called after the attachment was changed, be aware of the potential of recursion if you intend on calling {@link #setAttached} on the target.
+	 * </p>
```

Comment:
```suggestion
	 * <p>The event is called after the attachment was changed, be aware of the potential of recursion if you intend on calling {@link #setAttached} on the target.
```
Nit: this should follow the usual javadoc format